tutorials/building-a-simple-web-blog-with-vweb.md

```diff
@@ -346,7 +347,8 @@ pub fn (mut app App) new_article() vweb.Result {
 	sql app.db {
 		insert article into Article
 	}
-	return app.vweb.redirect('/article/')
```

Comment:
This is correct I think, since redirect also returns vweb.Result .
It may need to be: `return app.vweb.redirect('/')`.

Reply:
But when I run the code, got:
```
./blog.v:60:18: error: cannot use `void` as type `vweb.Result` in return argument 
   58 |         insert article into Article
   59 |     }
   60 |     return app.vweb.redirect('/')
      |                     ~~~~~~~~~~~~~
   61 |     // return vweb.Result{}
   62 | }
```
I checked the file `vweb.v`:
```
pub fn (mut ctx Context) redirect(url string) Result {
```
Why I got the error like that?

Reply:
I see. My vlang version is 0.1.28.1, which released at 28 Jun. In this version, `vweb.Context.redirect(string)` return `void`.